onnxruntime/contrib_ops/cuda/cuda_contrib_kernels.cc

```diff
@@ -133,124 +138,130 @@ KernelCreateInfo BuildKernelCreateInfo<void>() {
 
 Status RegisterCudaContribKernels(KernelRegistry& kernel_registry) {
   static const BuildKernelCreateInfoFn function_table[] = {
-      BuildKernelCreateInfo<void>,  // default entry to avoid the list become empty after ops-reducing
```

Comment:
Looks like lot of unrelated formatting changes.

Reply:
yes it is auto formatted.